chalice/app.py

```diff
@@ -202,7 +202,7 @@ def maybe_fake_504() -> bool:
             with flask_app.test_request_context(
                     path=path,
                     base_url="https://{}".format(app.current_request.headers["host"]),
-                    query_string=app.current_request.query_params,
+                    query_string=list((app.current_request.query_params or dict()).items()),
```

Comment:
why the change here?

Reply:
Needed to work with the new version of chalice.